stdlib/public/core/UnsafePointer.swift

```diff
@@ -304,11 +388,8 @@ public struct UnsafePointer<Pointee>: _Pointer {
   ///     the return value for the `withMemoryRebound(to:capacity:_:)` method.
   ///   - pointer: The pointer temporarily bound to `T`.
   /// - Returns: The return value, if any, of the `body` closure parameter.
-  @inlinable
   @_alwaysEmitIntoClient
-  // This custom silgen name is chosen to not interfere with the old ABI
-  @_silgen_name("_swift_se0333_UnsafePointer_withMemoryRebound")
```

Comment:
Haha this is cute.

Reply:
It's nice that it doesn't clash anymore!

Reply:
✨A kitten purrs for the first time every time a `@_silgen_name` gets removed✨

Reply:
(AFAIK symbols for @_aeic functions can also never clash with exported ones! But that could've been mistaken.)

Reply:
I had added these because there were errors otherwise. Glad to see them go.